ledger/block/src/lib.rs

```diff
@@ -77,10 +77,14 @@ pub struct Block<N: Network> {
     ratifications: Ratifications<N>,
     /// The solutions in the block.
     solutions: Solutions<N>,
+    /// The IDs of both accepted and aborted solutions in the previous block.
+    prior_solution_ids: Vec<SolutionID<N>>,
     /// The aborted solution IDs in this block.
     aborted_solution_ids: Vec<SolutionID<N>>,
     /// The transactions in this block.
     transactions: Transactions<N>,
+    /// The IDs of both accepted and aborted transactions in the previous block.
+    prior_transaction_ids: Vec<N::TransactionID>,
```

Comment:
I think `prior_*` is a bit too ambiguous. Our current usage of `aborted_or_existing_transaction_ids` is also quite verbose, but I'd still lean more towards verbosity for the sake of clarity.

Other options could be:
`existing_*`
`past_*`

Reply:
Also for backwards compatibility, an `Option<Vec<...>>` would be better than just an empty Vec for 2 reasons.

1. Services (wallets and explorers) have already serialized blocks without this field and might break if this is a required field.
2. Previous blocks will have an empty vec here; however, in reality they might actually have transactions or solutions that belong there based on the subdag state.